### PR TITLE
dashboard site refactor and staging deployment 

### LINKED ISSUE #114: setup staging environment for website-dashboard
**Objectives :-**

- Better folder structure for our project
image
- Helps to deploy our project on Heroku 
https://test-website-dashboard.herokuapp.com

### PR DESCRIPTION
**Pull request by Pavangbhat**
closes #114 
We are now serving our static files from server.js using express from **/public** folder 

**Objectives :-**

- Better folder structure for our project
image
- Helps to deploy our project on Heroku 
https://test-website-dashboard.herokuapp.com

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +21,11 @@
   "pre-commit": "check",
   "homepage": "https://github.com/Real-Dev-Squad/website-dashboard#readme",
   "devDependencies": {
+    "nodemon": "^2.0.15",
```
Comment (swarajpure) [inline package.json:24]: Not using nodemon in any script? 🤔

File: server.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('public'));
+
+app.get('/', (req, res) => {
+  res.render('index.html');
+});
+
```
Comment (swarajpure) [inline server.js:10]: What happens if I directly enter `/tasks` in my browser?

Comment (swarajpure) [reply server.js:10]: Or whatever is the path for the task creation form

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+*,
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
```
Comment (swarajpure) [inline src/style.css:9]: let's also add `padding: 0` as part of resetting browser's default CSS?

File: server.js (hunk 2 above)
Comment (Pavangbhat) [reply server.js:10]: For task creation it's **/task/index.html** and we directly navigate to task creation page 
https://test-website-dashboard.herokuapp.com/task/index.html

**Review by swarajpure — APPROVED** [review]
Looks good to me!

**Pull request merged**